src/experts/TimeInput.js

```diff
@@ -53,7 +50,7 @@
 			$.proxy( this._onRotatorChange, this ),
 			function() {
 				var value = self.viewState().value();
-				return value && value.getValue().calendar();
+				return value && value.getOption( 'calendarName' );
```

Comment:
This is the line that confused me, not the one above. There is an option `calendarModel` now. It contains an URI. I think this line needs to be `value.getOption( 'calendarModel' )`. The rotator changed from using string keys to URIs, right?